src/core/providers/qgsprovidersublayerdetails.h

```diff
@@ -105,6 +105,12 @@ class CORE_EXPORT QgsProviderSublayerDetails
 
       //! Set to TRUE if the default layer style should be loaded
       bool loadDefaultStyle = true;
+
+      /**
+       *  Set to TRUE if the statistics calculation should be skipped
+       *  \since QGIS 3.26
+       */
+      bool skipStatisticsCalculation = false;
```

Comment:
This flag does not seem to belong here... QgsPointCloudLayer::LayerOptions seems like more appropriate place.

Reply:
It is in both places since options are needed in order not to start the statistics calculation.
This is used only here: https://github.com/NEDJIMAbelgacem/QGIS/blob/ba0deaaa76a3891128e2d4a9b166ff067e97ea69/tests/src/providers/testqgscopcprovider.cpp#L199